scripts/set-react-version.mjs

```diff
@@ -97,6 +97,20 @@ async function checkEnvironment() {
   return true;
 }
 
+/**
+ * Disables [Jetifier](https://developer.android.com/tools/jetifier).
```

Comment:
might be worth adding an extra line to reference why from 74 onward we don't want to use jetifier anymore